numalogic/registry/mlflow_registry.py

```diff
@@ -122,36 +128,52 @@ def load(
                 return {}
             _LOGGER.info("Successfully loaded model %s from Mlflow", model_key)
             metadata = None
+            secondary_artifact = None
             model_properties = self.client.get_latest_versions(model_key, stages=["Production"])[-1]
             if model_properties.run_id:
                 run_id = model_properties.run_id
                 run_data = self.client.get_run(run_id).data.to_dictionary()
                 if run_data["params"]:
                     data = run_data["params"]
+                    secondary_artifact = pickle.loads(
```

Comment:
Is secondary artifact only one artifact? Or can it contain multiple?

Reply:
Yes. It can store only one artifact. It can be a scikit pipeline or even a single model too.

Reply:
Logically, it can store multiple artifacts too but only if they are passed as a single consolidated object (like Pipeline with multiple steps).